scripts/setup_genome.sh

```diff
@@ -0,0 +1,16 @@
+# This script will download, index, and prep reference genomes.
+
```

Comment:
We should reconfigure this script to allow for local testing.

Reply:
We'll figure out a way to provide genome files. I'll add this to the to-do list for project.